grml-debootstrap

```diff
@@ -613,6 +645,27 @@ case "${RELEASE}" in
     ;;
 esac
 
+if [ -z "$IMAGESIZE" ]; then
+  if [ "$RPI" ]; then
+    IMAGESIZE="3G"
+  else
+    IMAGESIZE="2G"
+  fi
+fi
+
+# RPi images *require* non-free-firmware as the raspi-firmware package resides
+# there, and Raspberry Pis are not bootable without it.
+if [ -z "$_opt_non_free" ] && [ "$RPI" ]; then
```

Comment:
i think this will need a warning, or better it should abort if non-free(-firmware) was not requested

Reply:
Currently there isn't an option for just enabling non-free-firmware but not the entirety of non-free. Should that be implemented, or is the job `--non-free` already does good enough?

Reply:
Added an error message with bailout if `--non-free` isn't specified. If we want something that only enables non-free-firmware when possible, I'd be happy to add that too.